lib/Cro/HTTP/Router.pm6

```diff
@@ -27,6 +27,18 @@ class X::Cro::HTTP::Router::NoRequestBodyMatch is Exception {
     }
 }
 
+class X::Cro::HTTP::Router::ConfusedCapture is Exception {
+    has $.body;
+    has $.content-type;
+    has $.signature;
+
+    method message() {
+        "The message body was parsed into '$!body.raku()' of type ($!body.^name()), " ~
```

Comment:
I think `$!body.raku` could end up huge; I think maybe let's:

1. Move it to the end after of the message
2. Length limit it (500 chars or so) and add "..."